snuba/migrations/snuba_migrations/profiles/0002_disable_vertical_merge_algorithm.py

```diff
@@ -0,0 +1,31 @@
+from typing import Sequence
+
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Use the horizontal merge algorithm
+    """
+
+    blocking = False
+
+    def forwards_local(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.RunSql(
+                storage_set=StorageSetKey.PROFILES,
+                statement=(
+                    "ALTER TABLE profiles_local MODIFY SETTINGS enable_vertical_merge_algorithm=0"
+                ),
+            )
+        ]
+
+    def backwards_local(self) -> Sequence[operations.SqlOperation]:
+        return []
```

Comment:
Please add the backwards migration:
`ALTER TABLE RESET SETTING enable_vertical_merge_algorithm`